beacon-chain/rpc/eth/validator/handlers.go

```diff
@@ -92,3 +79,31 @@ func (s *Server) GetAggregateAttestation(w http.ResponseWriter, r *http.Request)
 		}}
 	network.WriteJson(w, response)
 }
+
+// SubmitContributionAndProofs publishes multiple signed sync committee contribution and proofs.
+func (s *Server) SubmitContributionAndProofs(w http.ResponseWriter, r *http.Request) {
+	var req SubmitContributionAndProofsRequest
+	if r.Body != http.NoBody {
```

Comment:
does this also need that synced check for some other endpoints like publish block?

Reply:
no